assets/sass/utilities/_margin.scss

```diff
@@ -22,6 +22,10 @@
 		margin-top: 0 !important;
 	}
 
+	.googlesitekit-margin-top-1 {
+		margin-top: 1rem !important;
```

Comment:
Usually when adding `!important` it's good to add a reason _why_ we're using important. It helps with debugging/removals/auditing in the future.

I suspect the reason here is just to make sure these styles are always applied regardless of other classes—if that's the case I can add a comment for all of them.

Reply:
Yeah it's following the precedent here (which is inconsistent too), but if we're using a utility, it should be authoritative.